### DISCUSSION TITLE
Pnp Results Hover Card not working in Viva Connections

### DISCUSSION THREAD

**Discussion by AHast14** [microsoft-search/pnp-modern-search#2773] [Q&A]
Hello, i have set up a custom search using Pnp which works fine when on a browser. However, we have the site set as part of Viva Connections so users can access it via Teams. The persona hover card feature does not seem to work when inside Teams. Is this a known issue or is there a solution?

**Comment by kasperbolarsen** [comment]
Good find 👌 I haven't tried that yet, but will have a look at it next week. 
If you find a solution please let us know.

**Comment by kasperbolarsen** [comment]
Just added a PnP Search Results web part to my root site / intranet home which is available on Teams though Viva Connect. The Hover cards shows up in Teams as well as in SharePoint - in the browser.

Tested in the client as well - works just fine    , however my user is a GA, so it might have something to do with permissions?

**Comment by AHast14** [comment]
Interesting, I have tried accessing via Teams on both browser and client via both a standard and admin (not GA though) account and the error persists. Could additional permission be required in Graph API?

**Comment by kasperbolarsen** [comment]
So the Hover Card doesn't show up? Hmmm, any hint in the developer tools/F12 ?

**Comment by AHast14** [comment]
I cant see anything obvious but I'm not the best with developer tools. Is there a way I could share with yourself?

**Comment by AHast14** [comment]
Just wondering if there was any further thoughts on this as im out of ideas!
